cmd/artesia/app/config.go

```diff
@@ -1,13 +1,52 @@
 package app
 
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/viper"
+)
+
 // Config contains config data for the server
 type Config struct {
 	ListenAddress string
+	Database      DatabaseConfig
+}
+
+// DatabaseConfig has the data needed to connect to a Postgres DB
+type DatabaseConfig struct {
+	Host     string
+	DBName   string
+	User     string
+	Password string
+	Port     int
 }
 
-// NewDefaultConfig returns a default config
-func NewDefaultConfig() *Config {
```

Comment:
Should we bring back a default config? If I think about it probably not, but just putting this down.

Reply:
I think we'll have a better idea of this once we have more things that need configging, like SMTP or secret keys and whatnot. So yea, gonna leave this for later us to think about.

Reply:
:+1: